community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -316,9 +316,11 @@ private static String defaultPageCacheMemory()
                 long physicalMemory = (long) getTotalPhysicalMemorySize.invoke( os );
                 if ( 0 < physicalMemory && physicalMemory < Long.MAX_VALUE && maxHeapMemory < physicalMemory )
                 {
-                    long heuristic = (long) ((physicalMemory - maxHeapMemory) * 0.75);
-                    long min = 32 * 1024 * 1024; // We'd like at least 32 MiBs.
-                    long max = 1024 * 1024 * 1024 * 1024L; // Don't heuristically take more than 1 TiB.
+                    double ratioOfFreeMem = 0.50;
+                    long heuristic = (long) ((physicalMemory - maxHeapMemory) * ratioOfFreeMem);
+                    long MiB = 1024 * 1024;
```

Comment:
nope, KB is Settings unrelated in a same way as format, there is no way setting can influence that :)

Personally would prefer to have those constants in FileUtils, but that can be also influence of commons-io.